config/migrations/20210512164047-climate-subsidy/20210512164047-replace-ext-namespaces.sparql

```diff
@@ -0,0 +1,53 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX sbc: <http://data.lblod.info/vocabularies/subsidie/climate/>
+
+# Delets all instances of ext for the climate table
+
+DELETE {
+  GRAPH ?g {
+    ?sub ext:actionDescription ?desc;
+          ext:toRealiseUnits ?tru;
+          ext:costPerUnit ?cpu;
+          ext:restitution ?rtn;
+          ext:amountPerAction ?apa.
+  }
+} 
+INSERT {
+  GRAPH ?g {
+    ?sub sbc:actionDescription ?desc;
+          sbc:toRealiseUnits ?tru;
+          sbc:costPerUnit ?cpu;
+          sbc:restitution ?rtn;
+          sbc:amountPerAction ?apa.
+    }
+} 
+WHERE {
+  GRAPH ?g {
+    ?sub ext:actionDescription ?desc;
```

Comment:
This will not work for partially inserted data. To be on the safe side, you should make optional statements

Reply:
same is valid for the full where block

Reply:
Wrapped in an optional :)